data/src/main/java/com/android254/data/repos/OrganizersSource.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2022 DroidconKE
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.android254.data.repos
+
+import com.android254.data.dao.OrganizersDao
+import com.android254.data.network.apis.OrganizersApi
+import com.android254.data.util.OrganizerDataToDomainMapper.toDomain
+import com.android254.data.util.OrganizerDomainToEntityMapper.toDomain
+import com.android254.data.util.OrganizerDomainToEntityMapper.toEntity
+import com.android254.domain.repos.OrganizersRepository
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+class OrganizersSource @Inject constructor(
```

Comment:
Ideally we need to aggregate data in the repository

Reply:
Clarification on this?